CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Unreleased
-
+## 2.3.2 - 2023-05-30
```

Comment:
I suggest you increment to 2.4.0 since this is not a bug fix, it's an introduction of a new feature. After that, we may merge and tag the repo so that we have this version visible to Jenkins as a new tag. Eventually, this new tag should be built in Jenkins and the respective olive updated. Please let me know if anything unclear

Reply:
Added version tag 2.4.0